setup.cfg

```diff
@@ -5,8 +5,8 @@ description = Some out-of-the-box hooks for pre-commit.
 long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/pre-commit/pre-commit-hooks
-author = Anthony Sottile
-author_email = asottile@umich.edu
+author = Anthony Sottile; Alvaro Rodríguez Scelza - Loaderon at your service
+author_email = asottile@umich.edu; alvaroscelza@gmail.com
```

Comment:
when contributing to someone else's library you shouldn't modify author information